content/docs/12-enterprise-version/Helm Chart Install References

```diff
@@ -0,0 +1,161 @@
+| --- | --- | --- | --- | --- |
+| `stableEndpointAccess` | `False` | Boolean | Specifies whether EKS cluster is deployed in private network access through a stable private endpoint, if true, through a public endpoint. | Optional|
+
+## Registry and SCAR parameters
+
+**Registry and SCAR (Spectro Cloud Admission Registrar)** parameters are mandatory parameters in the **values.yaml** for installing **Palette** using Helm Chart.
+
+| **Parameters**       | **Type**   | **Description**                                             | **Required/Optional** |
+| ----------------- | ------ | -------------------------------------------------------- | ------------------ |
+| `registry.endpoint` | String | The endpoint URL for the registry.                      | Required           |
+| `registry.name`     | String | The name of the registry.                                | Required           |
+| `registry.password` | String | The password for the registry.                           | Required           |
+| `registry.username` | String | The username for the registry.                           | Required           |
+| `scar.endpoint`     | String | The endpoint URL of the Spectro Cloud Admission Registrar (SCAR). | Required           |
+| `scar.username`     | String | The username for the SCAR.                               | Required           |
+| `scar.password`     | String | The password for the SCAR.                               | Required           |
+
+To learn about the default values of the parameters, please contact Spectro Cloud sales.
+
+## Network Address Translation(NATS) parameters
+
+This section explains the parameters for the configuration file containing settings related to the **Network Address Translation (NATS)** messaging system. 
+
+| **Parameters **  | **Default Value** | **Type **   | **Description**                                                                                                               | **Required/Optional** |
+| ------------ | ------------- | ------- | ------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| `internal`   | `true`        | Boolean | Specifies if NATS shares the ingress load balancer or uses hostNetwork if true otherwise with a cloud load balancer.      | Optional           |
+| `natsUrl`    | `True`        | String  | Comma-separated list of <dns_name:port> mappings for nats load balancer service. Mandatory if nats.internal == false.     | Required           |
+| `Annotations`| `{}`          | Map     | NATS load balancer annotations.                                                                                           | Optional           |
+| `routes`     | `[]`          | List    | NATS routes & credentials. It is required only if the system is in HA mode.                                               | Optional           |
```

Comment:
This description doesn't make sense. How are routes and credentials associated together? EIther the description needs work, or we need to investigate this parameter more.

Reply:
I'd get this clarified with Arvind tomorrow.